mbed_devices/mbed_devices.py

```diff
@@ -3,37 +3,53 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 """API for listing devices."""
-from typing import Iterable
+from typing import Tuple, List
 
 from mbed_targets.exceptions import MbedTargetsError
 
-from mbed_devices._internal.candidate_device import CandidateDevice
-from mbed_devices._internal.detect_candidate_devices import detect_candidate_devices
+from mbed_devices._internal.detect_candidate_devices import CandidateDevice, detect_candidate_devices
 from mbed_devices._internal.resolve_target import resolve_target
 from mbed_devices._internal.exceptions import NoTargetForCandidate
 
+from mbed_targets import MbedTarget
 from mbed_devices.device import Device
 from mbed_devices.exceptions import DeviceLookupFailed
 
 
-def get_connected_devices() -> Iterable[Device]:
-    """Returns Mbed Devices connected to host computer."""
-    devices = []
+def get_connected_devices() -> Tuple[List[Device], List[Device]]:
+    """Returns Mbed Devices connected to host computer.
+
+    Returns:
+        A tuple containing two lists, the first containing devices identified as Mbed Targets the second those devices
+        connected but not identified as Mbed Targets.
+
+    Raises:
+        DeviceLookupFailed: TODO: find out when.
```

Comment:
Is this a temporary todo before the pr is merged?

Reply:
Damn, I found out and forgot to update the comment. Ta